projects/notebooks/visualizations/rhods-notebooks-performance/store/parsers.py

```diff
@@ -258,7 +259,7 @@ def _parse_start_end_time(dirname):
 def _parse_regression_results(dirname):
     regression_results_file = dirname / "regression.json"
     if not regression_results_file.exists():
-        logging.info(f"{regression_results_file.name} does not exist, ignoring the parsing of the regression analyses results.")
+        logging.info(f"{regression_results_file} does not exist, no new regression results found.")
```

Comment:
can you remove this line (or turn it into a `debug` call)?
or perhaps move it after the `return None` and turn it positive

as it is (from my code), it's too chatty and does not help much